src/Symfony/Component/Debug/Tests/DebugClassLoaderTest.php

```diff
@@ -195,6 +195,28 @@ class_exists('Test\\'.__NAMESPACE__.'\\'.$class, true);
         $this->assertSame($xError, $lastError);
     }
 
+    public function testInterfaceExtendsDeprecatedInterface()
+    {
+        set_error_handler('var_dump', 0);
+        $e = error_reporting(0);
+        trigger_error('', E_USER_DEPRECATED);
```

Comment:
@nicolas-grekas shouldn't this use ``E_USER_NOTICE`` here, as done in ``testDeprecatedSuperInSameNamespace`` ?